app-config.yaml

```diff
@@ -122,6 +122,13 @@ kafka:
         - localhost:9092
 
 integrations:
+  gcs:
+    - host: 'storage.cloud.google.com'
+      clientEmail:
+        $env: GCS_CLIENT_EMAIL
+      privateKey:
+        $env: GCS_PRIVATE_KEY
```

Comment:
Should update the config schema to reflect these https://github.com/backstage/backstage/blob/master/packages/integration/config.d.ts